.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
I'd prefer not to include these `.idea` configuration in the repo.

Reply:
Yes....agreed